src/views/partials/card.handlebars

```diff
@@ -0,0 +1,52 @@
+{{#each card}}
+<div class="card-modal" hidden>
+    <button class="close-button" id="closeModal">Close Modal</button>
+    <form id="updateCard" action="/board/card/{{boardId}}/{{_id}}" method="POST">
+        <h1>{{name}}</h1>
+        <h2>Members</h2>
+        {{#each members}}
+        {{#if assigned}}
```

Comment:
the assigned should be a checkbox input so you can select and unselect people and then submit the form to change who is assigned